src/ScoreBoard.js

```diff
@@ -36,17 +37,18 @@ class ScoreBoard extends React.Component {
       <section className="scoreboard">
         <div className="scoreboard-title">Score Board</div>
         <ol className="scoreboard-list">
-            {userMoves.map(({ name, moves }, index) => {
-              return (
-                <li
-                  key={index}
-                  className="scoreboard-item"
-                >
-                  <strong>{name}</strong>
-                  <span>: {moves} moves</span>
-                </li>
-              );
-            })}
+          {userMoves.map(({ name, moves }, index) => {
+            const finalMoves = moves * (11 - this.state.boardSize);
```

Comment:
destructure the boardSize state at the top of the renderScoreboard method as done here https://github.com/itaditya/trick-or-treat-game/blob/master/src/ScoreBoard.js#L34 

then you can directly do this here
```suggestion
            const finalMoves = moves * (11 - boardSize);
```

Reply:
Destructured.